distribution/ddf-common/src/main/resources/bin/setenv.bat

```diff
@@ -83,4 +83,22 @@ rem The Security Manager is turned off by default in DDF.
 rem
 rem set KARAF_SYSTEM_OPTS=-Dpolicy.provider=net.sourceforge.prograde.policy.ProGradePolicy -Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM -Djava.security.policy==%DDF_HOME%\security\default.policy -DproGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity
 
-set JAVA_OPTS=-Xms2g -Xmx4g -Dderby.system.home="%DDF_HOME%\data\derby" -Dderby.storage.fileSyncTransactionLog=true -Dfile.encoding=UTF8 -Dddf.home=%DDF_HOME% -Dddf.home.policy=%DDF_HOME_POLICY% -XX:+DisableAttachMechanism
\ No newline at end of file
+rem The following defines an environment variable referencing our script to be executed by the JVM
+rem when errors are detected. Unfortunately, forking the error process from Java does not expand
+rem variables as it does on Linux. Because the environment variable will be expanded by the batch
+rem script and not by Java, we are unable to specify the %p or any other arguments to the ddf_on_error.bat
+rem script as it requires injecting a space and variable expansions occuring in 2 places in karaf.bat
+rem renders it impossible to do so in a safe way no matter how we try to escape the space
+rem As a work around, we will have the ddf_on_error.bat script use the karaf.pid file that gets created
+rem and it will default to being invoked via the 'script'
+rem The space woud be important at the end of the DDF_ON_ERROR value to separate the last argument from
+rem the %p used by the JVM to represent the pid of the JVM
+rem set DDF_ON_ERROR=bin\ddf_on_error.bat script^ 
```

Comment:
❓ What is this `rem` line for?

Reply:
The actual way we would have like to set the env var but that didn't work with karaf.bat

Reply:
Since it doesn't work, shouldn't we just remove those lines to avoid any confusion?

Reply:
I left it there to go with the comment above so that someone would not try to do that. It also shows why it ends up being different than on the linux scripts.